cmd/fabric8-jenkins-idler/idler.go

```diff
@@ -46,15 +46,16 @@ func (idler *Idler) Run() {
 	oc := openshiftcontroller.NewOpenShiftController(
 		o,
 		t,
-		idler.config.GetConcurrentGroups(),
 		idler.config.GetIdleAfter(),
 		idler.config.GetFilteredNamespaces(),
 		idler.config.GetProxyURL(),
 		unidleRetry,
-		idler.config.GetUseWatch(),
 		idler.features,
 	)
 
+	//Spawn the main loop
+	oc.Run()
```

Comment:
Much better. Just a single _Run_ method. We can merge like this and once my pull request is in we can configure further to handle the go routines a bit better.